wgpu-core/src/device/mod.rs

```diff
@@ -2905,14 +2898,16 @@ impl<G: GlobalIdentityHandlerFactory> Global<G> {
             .change_replace(buffer_id, &ref_count, (), internal_use);
 
         device.lock_life(&mut token).map(buffer_id, ref_count);
+
+        Ok(())
     }
 
     pub fn buffer_get_mapped_range<B: GfxBackend>(
         &self,
         buffer_id: id::BufferId,
         offset: BufferAddress,
         _size: Option<BufferSize>,
```

Comment:
I just noticed this argument, what is it for? Is the intention to check that the mapped range has a valid size?

Reply:
we need to check and validate it, but we aren't using it in native implementation. A mapped range is just a pointer.
The parameter came from the WebGPU API spec, where it's needed to know how much to synchronize between content and GPU processes